shotgun_api3/shotgun.py

```diff
@@ -1473,20 +1494,33 @@ def update_project_last_accessed(self, project, user=None):
         record = self._call_rpc("update_project_last_accessed_by_current_user", params)
         result = self._parse_records(record)[0]
 
-
-    def _get_session_token(self):
-        """Hack to authenticate in order to download protected content
-        like Attachments
+    def generate_session_token(self):
```

Comment:
Is there a reason to call this generate_session_token rather than matching the server side call and calling it "get_session_token".  Get feels like it implies a Shotgun round trip where generate doesn't to me.